src/core/public/chrome/nav_links/nav_link.ts

```diff
@@ -142,12 +142,6 @@ export interface ChromeNavLink {
    * register an Application if needed.
    */
   readonly hidden?: boolean;
-
```

Comment:
Above, on the `href` property, can we now make that required?

Can we remove these properties: `subUrlBase`, `disableSubUrlTracking`, `linkToLastSubUrl`?